susemanager-utils/testing/docker/scripts/schema_migration_reportdb_test_pgsql.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -e
+
+echo "============================================================"
+echo "                      Migration test ReportDB               "
+echo "============================================================"
+
+cd /manager/susemanager-utils/testing/docker/scripts/
+
+# Move Postgres database to tmpfs to speed initialization and testing up
+if [ ! -z $PG_TMPFS_DIR ]; then
```

Comment:
Nitpicking here, but you can use `-n` instead of `! -z` and avoid the negated logic.

Reply:
we have this also in the other script. When we think about rewriting it into 1 script, we can think about it.
For now I would keep it.